css/main.css

```diff
@@ -172,11 +174,38 @@ body {
   text-align: center;
 }
 
-.join-description {
-  padding: 0 150px 32px;
+.desc-join {
+  color: var(--color-text-secondary);
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.desc-join p {
+  font-family: 'Roboto';
+  font-style: normal;
   font-weight: 400;
-  line-height: 175%;
-  text-align: justify;
+  font-size: 20px;
+  line-height: 34px;
+}
+
+.btn-join {
+  width: 22.5rem;
+  height: 5.25rem;
+  background: var(--color-button);
+  border-radius: 10px;
+  border: none;
+  font-family: 'Roboto';
+  font-style: normal;
+  font-weight: 700;
+  font-size: 36px;
+  line-height: 34px;
+  color: var(--color-text);
+}
+
+.btn-join:hover {
+  background: var(--color-button-hover);
+  cursor: pointer;
```

Comment:
Is the content responsive? Can we please have a screenshot of mobile and tablet views as well?

Reply:
Please watch the video and comment, if media queries are required or not
https://user-images.githubusercontent.com/70854507/211486574-d8c40972-9e37-4df6-a0a7-f16de6cc5b53.mp4

Reply:
added a some media queries for button please check

Reply:
Thanks